packages/emptyState/components/EmptyStateWithGraphic.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import { SpacingBox } from "../../styleUtils/modifiers";
+import { HeadingText2 } from "../../styleUtils/typography";
+import { EmptyStateProps } from "./EmptyState";
+import EmptyStateActions from "./EmptyStateActions";
+import { emptyStateGraphic, emptyStateWithGraphicBody } from "../style";
+
+interface GraphicDimensions {
+  width?: number;
+  height?: number;
+}
+
+interface EmptyStateWithGraphicProps extends EmptyStateProps {
+  /**
+   * The source of the image that appears in the empty state
+   */
+  graphicSrc: string;
+  /**
+   * The expected width and/or height dimensions of the graphic
+   */
+  graphicDimensions?: GraphicDimensions;
+}
+
+const EmptyStateWithGraphic: React.SFC<EmptyStateWithGraphicProps> = ({
+  graphicSrc,
+  graphicDimensions,
+  heading,
+  children,
+  primaryAction,
+  secondaryAction
+}) => {
+  const hasActions = primaryAction || secondaryAction;
+  return (
```

Comment:
Can we reuse `EmptyState` here?

Reply:
We could, but why would we?

Reply:
For consistency.

Reply:
There is a subtle (but intentional) difference with the spacing between the headline and the body content

Reply:
Okay, makes sense.